iroh-sync/src/store/fs.rs

```diff
@@ -149,11 +186,13 @@ impl Store {
     }
 
     /// Stores a new namespace
-    fn insert_namespace(&self, namespace: Namespace) -> Result<()> {
+    fn insert_namespace(&self, capability: Capability) -> Result<()> {
         let write_tx = self.db.begin_write()?;
         {
             let mut namespace_table = write_tx.open_table(NAMESPACES_TABLE)?;
-            namespace_table.insert(namespace.id().as_bytes(), &namespace.to_bytes())?;
+            let id = capability.id().to_bytes();
+            let (kind, bytes) = capability.raw();
+            namespace_table.insert(&id, (kind, &bytes))?;
```

Comment:
This would delete a namespace secret when importing a write capability for an existing read-only namespace, which is likely not the desired outcome.

Reply:
Fixed